content/docs/reference/technical-preview-free-tier.md

```diff
@@ -18,18 +18,20 @@ Technical Preview Free Tier users can create up to three projects in Neon. Proje
 Neon separates storage and compute and stores data in its own internal format.
 The 10GB data size limit applies to the logical size of a Neon project. The logical size is the sum of all relation sizes in the project.
 
-For users familiar with PostgreSQL, the logical size is roughly equal to:
+To check the logical size of the relations in your Neon project, run the following query:
```

Comment:
not sure every one of our users would be familiar with the word 'relations'

Reply:
Let's change that to databases!